### PR TITLE
Consistent Makefile between Medik8s opertators

### PR DESCRIPTION
**Pull request by razo7**
```release-note
Consistent Makefile between Medik8s opertators
```
It would be easier to see changes, commit after commit.
- reorder commands for make help
- add time fixes + update bundle
- update command name to test-e2e

### CODE REVIEW

**Review by beekhof — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -77,7 +88,7 @@ help: ## Display this help.
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
 
-generate: controller-gen protoc ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations. Also generate protoc / gRPC code
+generate: controller-gen protoc ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 	PATH=$(PATH):$(shell pwd)/bin/proto/bin && $(PROTOC) --go_out=. --go-grpc_out=. pkg/peerhealth/peerhealth.proto
```
Comment (razo7) [inline Makefile:93]: I have deleted the comment `Also generate protoc / gRPC code` at line 80 (which is line 91 in the updated file) which relates to this line (93). If this line is needed, thus I should return the deleted comment or delete this line if it is no longer needed

**Comment by razo7** [comment]
/hold
Waiting with the other PRs, till we finish with the next release. It is not urgent.

**Comment by razo7** [comment]
I am closing this PR and I will create a new one in [SNR](https://github.com/medik8s/self-node-remediation/pulls), since PP was updated to be SNR thus this repo won't be updated anymore (only backport IIUC).

**Pull request closed without merging**